# THIS SERVICE IS CURRENTLY USED FOR TESTING/DEBUGGING.
# PLEASE DO NOT USE THIS SERVICE EXTERNALLY.
# CONSTRAINTS CAN BE DIRECTLY SET BY THE SERVICES PROVIDED
# BY THE ENVIRONMENT SERVER.
# This service call updates the environment server
# with the set of constraints that need to be checked 
motion_planning_msgs/Constraints path_constraints
motion_planning_msgs/Constraints goal_constraints

---

# The update returns the constraints transformed to the frame in which 
# the planning monitor functions
motion_planning_msgs/Constraints path_constraints
motion_planning_msgs/Constraints goal_constraints
# True if the update was successful, false otherwise
motion_planning_msgs/ArmNavigationErrorCodes error_code